config/sync/core.entity_form_display.node.moj_pdf_item.default.yml

```diff
@@ -142,25 +188,25 @@ content:
     type: boolean_checkbox
     settings:
       display_label: true
-    weight: 15
+    weight: 13
```

Comment:
is weight like an ordering on the page kind of thing?

Reply:
Yep the weight controls the order of the fields on the form.  

All of these types of yml changes are generated by Drupal (i.e. not directly modified in code).  I've made changes to through the Drupal UI, and then ran the config export.  So this means you often get small changes like this that weren't entirely intentional (but aren't an issue other than it makes PR's difficult to review).

Reply:
oh right! that makes sense